src/plugins/sql-support/plugin.js

```diff
@@ -389,9 +389,14 @@ QueryBuilder.extend({
                 var opVal = sqlrl.call(this, value, data.operation);
                 if (operator == 'NOT LIKE') opVal.op = 'not_' + opVal.op;
 
+                var left_value = data.left.value.value;
```

Comment:
`data.left.value` is an object and `data.left.value.value` is a string with exact name